app/routes/_auth.tsx

```diff
@@ -0,0 +1,9 @@
+import { Outlet } from '@remix-run/react';
+
+export default function Component() {
+  return (
+    <div className="grow py-20 lg:py-40 bg-secondary w-full flex flex-col items-center justify-center">
+      <Outlet />
+    </div>
+  );
+}
```

Comment:
A Pathless Layout Route from Remix looks awesome! They don't add any segments to the URL.:rocket: